samples-js/functions/shopping.js

```diff
@@ -0,0 +1,47 @@
+const df = require("durable-functions");
+const { ShoppingCart, items } = require("./shoppingData");
+
+const entityName = "shoppingCartEntity";
+
+df.app.entity(entityName, function (context) {
+    const cart = context.df.getState(() => new ShoppingCart());
+
+    switch (context.df.operationName) {
+        case "ADD_ITEM":
```

Comment:
I thought the `ShoppingCart` example wasn't working due to a serialization issue. Did it get resolved?

Reply:
No it still doesn't work. Removed now.